pkgs/by-name/to/toolhive/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildGo124Module,
+  fetchFromGitHub,
+  testers,
+  toolhive,
+}:
+
+buildGo124Module rec {
```

```suggestion
buildGo124Module (finalAttrs: {
```
Go builder now supports this kinda lambdas, prefer them over `rec`